### PR TITLE
Edit target age

### PR DESCRIPTION
**Pull request by aslagle**
This connects to https://github.com/NYPL-Simplified/circulation-web/issues/31

Look at #149 first, it's in here too

### CODE REVIEW

**Comment by aslagle** [comment]
I wasn't sure how much validation to do here. In the admin interface, you only get the boxes to enter a target age range if you select children or ya as the audience.

File: api/admin/controller.py [hunk 1]
```diff
@@ -228,6 +230,33 @@ def edit(self, data_source, identifier):
...
+                if c.subject.type == Subject.AGE_RANGE:
+                    self._db.delete(c)
+
+            # Create a new classification with a high weight - higher than audience
+            age_range_identifier = "%s-%s" % (new_target_age_min, new_target_age_max)
+            work.primary_edition.primary_identifier.classify(
+                data_source=staff_data_source,
+                subject_type=Subject.AGE_RANGE,
+                subject_identifier=age_range_identifier,
+                weight=10000000,
```
Comment (leonardr) [inline api/admin/controller.py]: Maybe make this a constant since you'll be using this weight for genres and audience as well?

**Comment by leonardr** [comment]
:+1: once you move out that constant.

**Comment by aslagle** [comment]
Moved to a constant in this method for now. #148 is for consolidating the weight constants

**Pull request merged**